make/tools.mk

```diff
@@ -565,15 +565,14 @@ endif
 
 # OPENSSL download URL
 ifdef WINDOWS
-  openssl_install: OPENSSL_URL  := http://slproweb.com/download/Win32OpenSSL-1_0_2c.exe
+  openssl_install: OPENSSL_URL  := https://slproweb.com/download/Win32OpenSSL-1_0_2d.exe
```

Comment:
OK.  But I can't make this atomic / have the intermediate versions work.  The old version of the openssl_install stuff didn't work, and the 1_0_2c.exe doesn't exist anymore on the upstream site.  So there's no intermediate version of this that will work...